ros2topic/ros2topic/api/__init__.py

```diff
@@ -164,38 +166,39 @@ def qos_profile_from_short_keys(
 ) -> rclpy.qos.QoSProfile:
     """Construct a QoSProfile given the name of a preset, and optional overrides."""
     # Build a QoS profile based on user-supplied arguments
+    if not preset_profile:
+        preset_profile = default_preset_str
     profile = rclpy.qos.QoSPresetProfiles.get_from_short_key(preset_profile)
     if history:
         profile.history = rclpy.qos.QoSHistoryPolicy.get_from_short_key(history)
     if durability:
         profile.durability = rclpy.qos.QoSDurabilityPolicy.get_from_short_key(durability)
     if reliability:
         profile.reliability = rclpy.qos.QoSReliabilityPolicy.get_from_short_key(reliability)
-    if depth >= 0:
+    if depth and depth >= 0:
```

Comment:
Same here. I think we should handle the `None` case outside of this function. Alternatively, we should update the signature of this function:

```python
depth: Optional[int] = None
```

Reply:
Addressed on [160fce7](https://github.com/ros2/ros2cli/pull/613/commits/160fce7d4411ce53251d12ce92dd55b8e77fc046)